rio_tiler/io/xarray.py

```diff
@@ -127,8 +134,12 @@ def info(self) -> Info:
             "count": self.input.rio.count,
             "width": self.input.rio.width,
             "height": self.input.rio.height,
-            "attrs": self.input.attrs,
+            "attrs": {
+                k: (v.tolist() if isinstance(v, (numpy.ndarray, numpy.generic)) else v)
+                for k, v in self.input.attrs.items()
+            },
```

Comment:
There was a possible issue in titiler where pydantic wouldn't be able to serialize the `attrs` because pydantic do not handle well numpy arrays